src/AlgebraicGeometry/Schemes/AffineSchemes/Morphisms/Attributes.jl

```diff
@@ -251,15 +251,15 @@ Spectrum
 
 julia> f = inclusion_morphism(X, Y)
 Affine scheme morphism
-  from [x1, x2, x3]  V(x1)
+  from [x1, x2, x3]  scheme(x1)
   to   [x1, x2, x3]  affine 3-space over QQ
 given by the pullback function
   x1 -> 0
   x2 -> x2
   x3 -> x3
 
 julia> graph(f)
-(V(x1, -x1, x2 - x2, x3 - x3), Hom: V(x1, -x1, x2 - x2, x3 - x3) -> V(x1), Hom: V(x1, -x1, x2 - x2, x3 - x3) -> affine 3-space over QQ with coordinates [x1, x2, x3])
+(scheme(x1, -x1, x2 - x2, x3 - x3), Hom: scheme(x1, -x1, x2 - x2, x3 - x3) -> scheme(x1), Hom: scheme(x1, -x1, x2 - x2, x3 - x3) -> affine 3-space over QQ with coordinates [x1, x2, x3])
```

Comment:
'Hom' feels like the wrong term here. On the algebraic side, 'Hom' is the right word, but usually, this is not referred to as a homomorphism on the geometric side, but as a morphism. Is this printing tied to the printing on the algebraic side? If so, leave it as it is. If not, this needs to be discussed (unless it has already been discussed and decided).

Reply:
It is not tied to the algebraic side. As far as I know there is no executive decision. We should be free to switch to `Mor` but let me check back with someone else.